### PR TITLE
[SUPPORTENG-405] Update transfer.md

### PR DESCRIPTION
**Pull request by kgodtfredsen**
Clean this up a bit

### CODE REVIEW

File: docs/_docs/transfer.md [hunk 1]
```diff
@@ -104,20 +103,41 @@ When you’ve finished testing your Transfer-capable trigger and you want to mak
...
-Example of a trigger perform. This API uses the page/offset approach to paging. Here we check `bundle.meta.page` to differentiate handling for a Transfer request vs. a regular Zap handling which just returns the most recent objects.
+
+const getAthleteActivities = async (z, per_page, bundle) => {
+  const response = await z.request({
+    url: `${API_BASE_URL}/athlete/activities`,
+    params: {
+      per_page: per_page,
+      page: bundle.meta.page + 1, // Strava starts paging at 1
+    },
+  });
```
Comment (eliangcs) [inline docs/_docs/transfer.md:137]: Should we return a parsed result here?

```javascript
const getAthleteActivities = async (z, per_page, bundle) => {
  const response = await z.request({
    // ...
  });
  return response.data;
};
```

**Review by eliangcs — APPROVED** [review]
Looks good to me, except for the missing `return` statement I mentioned in the comment. Do we need to get an eye from the Transfer team too, as I'm not super familiar with Transfer?

File: docs/_docs/transfer.md (hunk 1 above)
Comment (kgodtfredsen) [reply docs/_docs/transfer.md:137]: yes we should, where the heck did that go!

**Comment by kgodtfredsen** [comment]
> Looks good to me, except for the missing `return` statement I mentioned in the comment. Do we need to get an eye from the Transfer team too, as I'm not super familiar with Transfer?

Hye @eliangcs  - thanks for the catch and the comment. I checked with Dani in transferland to confirm if we needed any changes from what Zane first created to make it public and it looked OK. I'm going to wait to do the final edit and merge until we release the transfer scheduling feature.

**Review by eliangcs — DISMISSED** [review]

**Comment by kgodtfredsen** [comment]
Heya @eliangcs  - I ended having to do one more update here, as the Transfer team decided to keep the product as "beta", and that the whole 60 seconds per trigger invoke didn't take into consideration that lambda and python apps have a shorter timeout. I updated those sections back to 30 seconds, and then added a little more on how private apps look in the Transfer UI.

**Review by eliangcs — APPROVED** [review]

**Pull request merged**